zerver/openapi/zulip.yaml

```diff
@@ -20463,7 +20463,30 @@ paths:
                 contentType: application/json
       responses:
         "200":
-          $ref: "#/components/responses/SimpleSuccess"
+          description: |
+            A success response containing the unique ID of the user group
+            with the key `group_id`. This field provides a straightforward
+            way to reference the newly created user group.
```

Comment:
Generally you don't need to mention the name of the key here; that's in the generated documentation